### ISSUE TITLE
Citizens Sentinels attack shulker portcullis

### ISSUE THREAD

**Issue by BorealPawPads** [bonn2/BigDoorsPhysics#2]
I'm submitting this issue because of what it says on the changelog.

>NPE in ShulkerPacketEditor (fixes Citizens compatibility) Thanks @repeater64 for the report

However, when using shulker as the method instead of barriers, Sentinel-class NPCs will still attack the doors. This is true even if spawn-shulkers-on-door is set to false.

When using barrier, the problem is not there, however then the physics aren't as neat.

**Comment by bonn2** [comment]
That fix was for a different Citzens issue (that caused the plugin to crash altogether).
As for `spawn-shulkers-on-door` that is intended to fix issues where mobs are only allowed to spawn in specific regions.

Anyways I'll have to look into if Citizens has a way to designate mobs as "friendly" to the sentinels.

**Comment by repeater64** [comment]
Users can set up the sentinel trait to ignore certain mobs like shulkers, so @BorealPawPads can do this

**Comment by bonn2** [comment]
Yeah, I saw that. I was hoping there would be some tag I could put on them to make them be ignored by default.  However that should be a good for-now workaround

**Comment by BorealPawPads** [comment]
I can confirm that setting Sentinels to ignore shulkers works. (That being said, it's a pain cause you have to edit each NPC manually.)

So I suppose this can be marked as resolved or maybe even not-a-bug since the sentinels -are- attacking an enemy as it spawns!
